samples/msal-browser-samples/ChromiumExtensionSample/README.md

```diff
@@ -20,7 +20,7 @@ This folder contains a sample Chromium extensions demonstrating how to integrate
 
 ## MSAL Usage
 
-Chromium extensions are unable to perform certain types of navigation, so applications should leverage the [`chrome.identity.launchWebAuthFlow`](https://developer.chrome.com/apps/identity#method-launchWebAuthFlow) API to perform interactive auth requests. This API takes a url to navigate to, and a callback that will be invoked once the auth flow is completed (which is signaled by AAD redirecting back to the `chromiumapp.com` url mentioned earlier). Chromium extensions using MSAL Browser can build working auth flows by composing the MSAL `loginRedirect/acquireTokenRedirect` and `handleRedirectPromise` APIs to generate a url and handle the response:
+Chromium extensions are unable to perform certain types of navigation, so applications should leverage the [`chrome.identity.launchWebAuthFlow`](https://developer.chrome.com/apps/identity#method-launchWebAuthFlow) API to perform interactive auth requests. This API takes a url to navigate to, and a callback that will be invoked once the auth flow is completed (which is signaled by ME-ID redirecting back to the `chromiumapp.com` url mentioned earlier). Chromium extensions using MSAL Browser can build working auth flows by composing the MSAL `loginRedirect/acquireTokenRedirect` and `handleRedirectPromise` APIs to generate a url and handle the response:
```

Comment:
Should we go with Microsoft Entra ID here? In isolation some people may not know what this means.

Reply:
You are exactly right! This is something I've been struggling with in the script. We don't want ME-ID there at all except for outlier cases where there's some kind of space constraint. Thank you!